bzl/bazel.rc

```diff
@@ -25,6 +25,9 @@ build --action_env=HTTPS_PROXY
 build --action_env=no_proxy
 build --action_env=http_proxy
 build --action_env=https_proxy
+build --action_env=XLA_DUMPDIR="/tmp/hlo_module"
```

Comment:
Could you keep at the TF-specific changes together in this file? We might do the integration with TF different in the future.